M2/INSTALL

```diff
@@ -174,7 +174,7 @@ various modern operating systems:
       Starting with Mac OS X version 10.14 (Mojave), /usr/include needs to be created with
         this command:
 	  sudo installer -pkg macOS_SDK_headers_for_macOS_10.14.pkg -target /
-	That file is found in the directory
+	The pkg file is found in the directory
```

Comment:
"pkg" is not an English word.

Reply:
Hmm, OK, but the formulation with "that file" confused me.